backend/src/director.py

```diff
@@ -10,9 +11,13 @@
 director_prompt = engine.load_prompt("director")
 determine_intention_prompt = engine.load_prompt("determine-intention")
 
+
 def question(question):
     logging.debug("Received utterance: {question}")
 
+    intent = intent_agent.invoke(question)
+    logging.debug(f"Intent determined: {intent}")
+
     if determine_intention(question) == "TRUE":
         task_dict = create_tasks(question, agents_details)
```

Comment:
Are we still calling this tasks?

Reply:
yes as that is unchanged at the moment. When we do the wire up ticket, then we can change the naming to questions and intent (maybe this question method itself should change too)